tools/ci_import

```diff
@@ -0,0 +1,99 @@
+#!/bin/env python
+
+import argparse
+import json
+import os
+import sys
+import zipfile
+import tempfile
+from os import path
+
+import gitlab
+
+IMI_FOLDER = "generated-image-infos"
+MANIFEST_DB_PROJECT_ID = 36844106
+
+
+def import_image_infos(pipeline, token, verbose=False):
+    success = True
+    gl = gitlab.Gitlab("https://gitlab.com", oauth_token=token)
+    project = gl.projects.get(MANIFEST_DB_PROJECT_ID)
+    pipeline = project.pipelines.get(pipeline)
+    jobs = pipeline.jobs.list(all=True)
+    for job in jobs:
+        if verbose:
+            print(f"Job {job.get_id()}:")
+        directory = path.join(tempfile.TemporaryDirectory().name, str(job.get_id()))
+        os.makedirs(directory, exist_ok=True)
+        filename = os.path.join(directory, "artifact.zip")
+        if verbose:
+            print(f"    downloading artifact.zip to {filename} for job {job.get_id()}")
+        try:
```

Comment:
I think that this `try` block is too long. Can we put only the following line into it instead of the whole image updating logic that cannot throw `gitlab.exceptions.GitlabGetError`?
```python3
project.jobs.get(job.get_id()).artifacts(streamed=True, action=f.write)
```

Reply:
An even better solution might be to put this block:

```python3
        filename = os.path.join(directory, "artifact.zip")
        if verbose:
            print(f"    downloading artifact.zip to {filename} for job {job.get_id()}")
        try:
            with open(filename, "wb") as f:
                project.jobs.get(job.get_id()).artifacts(streamed=True, action=f.write)
            z = zipfile.ZipFile(filename)
            z.extractall(directory)
            if verbose:
                print(f"    image information imported for {job.get_id()}")
            try:
                imis = os.listdir(os.path.join(directory, IMI_FOLDER))
            except FileNotFoundError:
                continue
```

into a separate function names something like `load_imis_from_gitlab_job`. It can return false if `artifact.zip` doesn't exist, or `IMI_FOLDER` inside the archive doesn't exist.

Reply:
I've done an other kind of refactor, but keeping the idea of simplifying this part.